projects/ngx-resource-calendar/src/lib/resource-calendar.component.ts

```diff
@@ -296,7 +312,9 @@ export class ResourceCalendarComponent implements OnChanges {
       return {
         data: event,
         position: this.calculatePosition(event, day),
-        height: this.calculateHeight(event)
+        height: this.calculateHeight(event),
+        left: event.left || 0,
+        width: event.width || 100
```

Comment:
Vähän ehkä epäoptimi käytettävyyden kannalta että korkeus on pikseleinä ja left/width on prosentteina. Pitää Vaan Tietää ™️. Olisko mahdollista että näissä muuttujissa annettais suoraan arvoiks esim. "200px" tai "50%" ja käytettäis niitä tuolla viewin puolella?